configs/config.toml

```diff
@@ -25,3 +25,11 @@
   [Agent.server]
     nats_url = "localhost:4222"
     port = "9000"
+  
+  # interval - interval in seconds in which heartbeat is expected
+  [Agent.heartbeat]
+    interval = 30s
```

Comment:
Is this working without `"30s"`?

Reply:
we are using `time.ParseDuration` like here https://github.com/mainflux/agent/pull/43#discussion_r438118724

Reply:
Could we replace `30s` by `"30s"`?